src/goto-programs/remove_const_function_pointers.h

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************\
+
+Module: Goto Programs
+
+Author: Thomas Kiley, thomas.kiley@diffblue.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_REMOVE_CONST_FUNCTION_POINTERS_H
+#define CPROVER_GOTO_PROGRAMS_REMOVE_CONST_FUNCTION_POINTERS_H
+
+#include "goto_model.h"
+#include <util/message.h>
+#include <util/expr.h>
+#include <util/namespace.h>
+
+
+class remove_const_function_pointerst:public messaget
+{
+public:
+  typedef std::set<exprt> functionst;
```

Comment:
I'd suggestion one of two options here: 1) the tests really shouldn't be depending on ordering, thus try to adjust the tests; 2) or add a comment saying that using `std::set` simplifies testing.